functions/exercise22/scriptSpec.js

```diff
@@ -0,0 +1,19 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals countNumberOccurrencesOfSpecifiedLetterWithinString*/
+
+
+describe("Count the number of occurrences of the specified letter within the string", (() => {
+
+    it("In the string the given letter occurs x times", (() => {
```

Comment:
This description is too generic. As you have a very simple input and output here, you may use it in description.